common/constants.go

```diff
@@ -86,12 +86,16 @@ const (
 )
 
 const (
-	// SystemDomainName is domain name for all cadence system workflows
+	// SystemGlobalDomainName is global domain name for cadence system workflows running globally
+	SystemGlobalDomainName = "cadence-system-global"
+	// SystemDomainName is domain name for cadence system workflows running in local cluster
 	SystemDomainName = "cadence-system"
```

Comment:
plz also change this to local

Reply:
that will be related to the existing domain we created in production. Let me create an issue and talk to the team.

Reply:
https://github.com/uber/cadence/issues/2000